opts/opts_test.go

```diff
@@ -127,8 +128,12 @@ func TestValidateExtraHosts(t *testing.T) {
 	}
 
 	for _, extrahost := range invalid {
-		if _, err := ValidateExtraHost(extrahost); err == nil {
-			t.Fatalf("ValidateExtraHost(`" + extrahost + "`) should have failed validation")
+		if _, err := ValidateExtraHost(extrahost[0]); err == nil {
+			t.Fatalf("ValidateExtraHost(`" + extrahost[0] + "`) should have failed validation")
+		} else {
+			if !strings.Contains(err.Error(), extrahost[1]) {
+				t.Fatalf("ValidateExtraHost(`"+extrahost[0]+"`) error should contain %q", extrahost[1])
```

Comment:
There's a weird formatting issue here: why aren't the `+` surround in spaces but the ones above are?